.htaccess

```diff
@@ -44,7 +45,6 @@ RewriteRule		^apple-touch-icon-precomposed.png$	-							[L,R=404]
 
 #top level sections
 RewriteRule		^history[/]*$					index.php?s=history				[NC,L]
-RewriteRule		^raceday[/]*$					index.php?s=raceday				[NC,L]
 RewriteRule		^search[/]*$					index.php?s=search				[NC,L]
 RewriteRule		^events[/]*$					index.php?s=events				[NC,L]
```

Comment:
You removed this file, so the rewrite rule should go too?

Likewise, you should remove the corresponding section from index.php